### PR TITLE
Updated ocular install instructions

### PR DESCRIPTION
**Pull request by NWeber-SL**
Please review carefully, especially step #6.

### CODE REVIEW

File: using-ocular/installation.md [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+8. If the ShiftLeft static policy already exists, you are given the option to delete it. Otherwise the installer unpacks the ShiftLeft static policy to `~/.shiftleft/policy/static`.
 
-The install script:
+A successful installation is indicated by this appearing in the terminal
 
-* asks you where to install ShiftLeft Ocular (defaults to `~/bin/ocular`).
-* checks if there is an existing installation, and if so, offers to delete it.
-* unpacks the ShiftLeft dynamic policy to `~/.shiftleft/policy/dynamic`, and if it already exists, offers to delete it.
-* unpacks the ShiftLeft static policy to `~/.shiftleft/policy/static`, and if it already exists, offers to delete it.
+![Ocular in terminal](img/ocular.jpg)
```
Comment (tuxology) [inline using-ocular/installation.md]: This is incorrect. This appears when Ocular runs. You can say. "Ocular can bow be started using `./ocular.sh` on Linux and `.\ocular.ps1` on Windows PowerShell. The image is not necessary

File: using-ocular/installation.md [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
 
-```bash
-unzip ocular-distribution.zip
+```
+unzip ocular-distribution-[version].zip
+```
+
+2. Enter the password you received from ShiftLeft. The directory `ocular-distribution` is created.
+
+3. Navigate to the new directory, using the command
```
[the remark this answers is not in the document]
Comment (NWeber-SL) [reply using-ocular/installation.md]: Done

File: using-ocular/installation.md [hunk 3]
```diff
@@ -1,28 +1,40 @@
...
+
+```
 cd ocular-distribution
 ```
 
-Run the installer and follow the prompts:
+4. In the `ocular-distribution` directory, run the installer using the following command. If you don't have permission, first use the  command `chmod +x install.sh` and then run the installer again.
```
[the remark this answers is not in the document]
Comment (NWeber-SL) [reply using-ocular/installation.md]: Added

File: using-ocular/installation.md [hunk 4]
```diff
@@ -1,28 +1,40 @@
...
 
-```bash
-bash ./install.sh
 ```
+./install.sh
+```
+
+5. Identify where you want to install ShiftLeft Ocular (defaults to C`~/bin/ocular`).
```
[the remark this answers is not in the document]
Comment (NWeber-SL) [reply using-ocular/installation.md]: Added

File: using-ocular/installation.md (hunk 1 above)
Comment (NWeber-SL) [reply using-ocular/installation.md]: Fixed

**Pull request merged**